frontend/src/components/rideshareEventForm/RideshareEventForm.jsx

```diff
@@ -0,0 +1,368 @@
+import Header from '../header/Header'
+import NearbyEventsMap from '../NearbyEventsMap'
+import axios from 'axios'
+import { useState, useEffect } from 'react'
+import { Menu, MenuButton, MenuItem, MenuItems } from '@headlessui/react'
+import { ChevronDownIcon } from '@heroicons/react/20/solid'
+
+function RideshareEventForm({ user }) {
+    const current_eventId = window.location.pathname.split('/')[3]
```

Comment:
https://github.com/CristobalMe/Event_Finder/pull/11#discussion_r1668906638

Reply:
<img width="342" alt="Screenshot 2024-07-23 at 2 40 02 PM" src="https://github.com/user-attachments/assets/6e7dc8b7-7639-4b1f-97a0-3053bf95aa97">
<img width="476" alt="Screenshot 2024-07-23 at 2 40 42 PM" src="https://github.com/user-attachments/assets/d8d6200c-1a33-4fbe-a0e7-2075a01dedd4">

I don't know how to implement it